core/templates/pages/license-page/license-page.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="oppia-license-page">
+  <div class="oppia-static-header">
+    <h1 [innerHTML]="'I18N_LICENSE_TERMS_HEADING' | translate"></h1>
+  </div>
+  <background-banner></background-banner>
+  <div>
+    <div class="oppia-static-content-below-banner">
+      <div class="oppia-page-card oppia-static-content">
+        <div class="about-tab-container">
+          <div class="oppia-license-terms oppia-about-tab-content"
+               [ngClass]="{'oppia-about-visible-content': true}">
+            <div class="oppia-static-card-content oppia-static-card-content-narrow">
+              <h2 [innerHTML]="'I18N_LICENSE_PAGE_LICENSE_HEADING' | translate"></h2>
+              <p [innerHTML]="'I18N_LICENSE_PAGE_PARAGRAPH_1' | translate"></p>
+              <p [innerHTML]="'I18N_LICENSE_PAGE_PARAGRAPH_2' | translate"></p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="oppia-footer-padding-below-banner">
+  </div>
```

Comment:
How is this used?

Reply:
This add space between license box and footer.

Reply:
Why can we use margin-bottom in the above container instead of creating a blank container to add space?

Reply:
done